File: Solution/Mundial.Infra/Model/Installment.cs
using System;

namespace Mundial.Infra.Model
{
    public class Installment: MundialModel
    {
        public int ServiceOrderId {get; set;}
        public int ProductId {get; set;}
        public Product Product {get; set;}
        public float OriginalValue {get; set;}
        public float NewValue {get; set;}
        public int DivisionNumber {get; set;}
        public float PaidValue {get; set;}
        public string Observation {get; set;}
    }
}

File: Client/mundial-client/src/app/models/installment.ts
import { MundialModel } from './abstract/mundial-model';
import { Product } from './product';

export class Installment extends MundialModel {
    public ServiceOrderId?: number;
    public ProductId?: number;
    public Product?: Product 
    public OriginalValue?: number;
    public NewValue?: number; 
    public DivisionNumber?: number; 
    public PaidValue?: number; 
    public Observation?: string;

    constructor(creationDate: Date,number:number,Id?: number, exclusionDate?: Date,
        serviceOrderID?: number, productId?:number, product?: Product,
        originalValue?:number, newValue?: number,divisionNumber?: number, 
        paidValue?:number,observation?:string )
    {
      super(creationDate, number, exclusionDate, Id);
      this.ServiceOrderId = serviceOrderID;
      this.ProductId = productId;
      this.Product = product;
      this.OriginalValue = originalValue;
      this.NewValue = newValue;
      this.DivisionNumber = divisionNumber;
      this.PaidValue = paidValue;
      this.Observation = observation;
    }
}
